.eslintrc.pre-push.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": ".eslintrc.json",
+    "rules": {
+        "no-debugger": "error",
+        "header/header": ["error", "config/license-header.js"],
+        "import/order": "error",
```

Comment:
Prościej będzie rozwiązywać konflikty w importach

Reply:
A to w takim przypadku ma to duży sens :D 
Jeszcze może console byśmy dali jako error. Niby teraz i tak nie da się jej spushować, bo nie można jeśli mamy warningi, ale byłoby to dodatkowe zabezpiecznie XD